README.md

```diff
@@ -22,11 +22,13 @@ curl -X POST http://localhost:9080/1.0/register -H "Content-Type: application/js
 '{ "options": { \
     "publicKeyBase58": "6hUdTYK8pp3h1EwwJ4j8afsHHgJf8qDvdLrKbHPJkw6x", \
     "extIds": ["test", "external", "ids"] \
+    "network": "test" \
 }'
 ```
 The options are:
 * `publicKeyBase58` - **REQUIRED**: the public key to be associated with the DID
 * `extIds` - **OPTIONAL**: the unique external ids for the chain entry forming the basis of the DID. If none are provided, `[SHA256(publicKey)]` is used.
+* `network` - **OPTIONAL**: should be either `"test"` or `"main"` and determines where the DID will be created. Default is `"main"`.
```

Comment:
Suggest to keep mainnet vs main and testnet vs test consistent everywhere.

Reply:
I think I got everything. Changed it to `testnet` and `mainnet` everywhere.

Reply:
You missed the options example request

Reply:
Ah yes, an important one. Updated.